demo/src/demo.ts

```diff
@@ -192,23 +197,24 @@ async function main() {
 
   // Step 4: Delegate creates a new Verifiable Document
   console.log(`\n❄️  Verifiable Document Creation `)
+  let callBackFn = async ({ data }) => ({
+    signature: delegateTwoKeys.authentication.sign(data),
+    keyType: delegateTwoKeys.authentication.type,
+    keyUri: `${delegateTwoDid.uri}${delegateTwoDid.authentication[0].id}`,
+  })
```

Comment:
notice that, the callback function was wrong in your earlier commits @adi-a11y. Otherwise all good.

Reply:
Oh! Understood. Thank you so much. 
I tried to create a VP using the old document and still getting the successful verification message!